docs/_pages/docs/03-understand/05-trade-escrow.md

```diff
@@ -2,10 +2,64 @@
+
+As soon as the order taker locks their bond, the buyer and seller are required to submit the payout invoice and trade escrow, respectively, within the given time limit.
+
+By default, the expiry timer is three hours; however, as the order maker, you can customize the timer to be anywhere from one hour to eight hours. In other words, modify the time allowed to lock the escrow funds and provide the payout invoice. Maybe you want an expedited transaction and will set the timer to a maximum of one hour instead of three hours.
+
+If you lock the escrow funds before the buyer has provided the payout invoice, then you will have to wait to chat with your peer until after they have provided the invoice.
+
+If you don't lock the escrow funds at all, then the order will expire and the seller forfeits their bond. Half of the lost bond goes to the honest robot as compensation for wasted time. Similarly, if the buyer does not provide the payout invoice within the given time limit, then the buyer forfeits their bond.
+
+After the trade escrow is locked, the order cannot be cancelled except if both the maker and taker agree to collaboratively cancel. Furthermore, after the seller confirms the fiat was received, the order cannot be collaboratively cancelled anymore. The order can then be either completed successfully or enter into a dispute.
+
+## **How and When the Escrow is Released**
+
+The trade escrow is always released to its rightful owner depending on the status of the trade or, if necessary, the outcome of the dispute. There are two scenarios that cause the trade escrow to be released:
+* Completing a successful trade (seller confirms the fiat was received)
+* Opening a dispute if the trade was unsuccessful (seller intentionally did not confirm the fiat was received)
+
+The scenarios above are expanded upon in additional detail below.
+
+Once the fiat payment method is coordinated with the buyer, the seller clicks "Confirm Fiat Received" to end the trade which releases the escrow funds to the buyer. The seller should only confirm the fiat is received *after* it appears in their possession.
+
+If you never received the fiat payment from the buyer, do not click "Confirm Fiat Received" and instead open a dispute for RoboSats staff to review. Trying to cheat by intentionally not confirming the fiat was received results in a dispute being automatically opened on the buyer's behalf.
+
+The cheating robot will risk losing that dispute and consequently forfeit their bond, where half of that bond is rewarded to the honest robot and the entirety of the locked escrow is released to the honest robot as well.
```

Comment:
Half of the bond is rewarded to your peer only if the bond is lost outside of a dispute. When a dispute takes place, the platform takes the full bond, reasons: 1) there should not be any incentives to open fraudulent disputes to earn yield 2) solving disputes is costly in human labor, the cost has to be covered with the bonds.